src/app/pokemonListSlice.tsx

```diff
@@ -3,28 +3,39 @@ import { PokemonTypeInitialState } from "../utils/Types";
 import { getInitialPokemonData } from "./getInitialData";
 import { getPokemonsData } from "./getPokemonData";
 import { getCurrentPokemonData } from "./getCurrentPokemonData";
+
 const initialState: PokemonTypeInitialState = {
   allPokemon: undefined,
```

Comment:
Set the default value of this property as empty array

```suggestion
  allPokemon:[],
```